ev3dev2/__init__.py

```diff
@@ -281,6 +287,15 @@ def get_attr_int(self, attribute, name):
         attribute, value = self._get_attribute(attribute, name)
         return attribute, int(value)
 
+    def get_cached_attr_int(self, filehandle, keyword):
+        value = self._attr_cache.get(keyword)
+
+        if value is None:
+            (filehandle, value) = self.get_attr_int(self._count_per_rot, keyword)
```

Comment:
Specifically `_count_per_rot`? 😕

Reply:
crap this is a huge cut-n-paste error...will fix all of these